src/vaadin-radio-group-mixin.ts

```diff
@@ -0,0 +1,330 @@
+import { LitElement, html, property, PropertyValues } from 'lit-element';
+import { Constructor } from '@vaadin/mixin-utils';
+import { DisabledStateMixin, DisabledStateInterface } from '@vaadin/disabled-state-mixin/disabled-state-mixin.js';
+import { FocusVisibleMixin, FocusVisibleInterface } from '@vaadin/focus-visible-mixin/focus-visible-mixin.js';
+import { FocusVisibleClass } from '@vaadin/focus-visible-mixin/focus-visible-class.js';
+import {
+  KeyboardDirectionMixin,
+  KeyboardDirectionInterface
+} from '@vaadin/keyboard-direction-mixin/keyboard-direction-mixin.js';
+import { KeyboardDirectionClass } from '@vaadin/keyboard-direction-mixin/keyboard-direction-class.js';
+import { RovingTabIndexMixin } from '@vaadin/roving-tabindex-mixin/roving-tabindex-mixin.js';
+import { SlottedItemsMixin, SlottedItemsInterface } from '@vaadin/slotted-items-mixin/slotted-items-mixin.js';
+import { RadioButtonMixinClass } from './vaadin-radio-button-mixin-class';
+import { RadioButton } from './vaadin-radio-button-mixin';
+
+export interface RadioGroupInterface {
+  value: string | null | undefined;
+
+  label: string | null | undefined;
+
+  invalid: boolean | null | undefined;
+
+  required: boolean | null | undefined;
+
+  readonly: boolean | null | undefined;
```

Comment:
There is a `DisabledStateMixin` and `DisabledStateInterface`.
Disabled and Readonly states are quite similar. Should we introduce `ReadonlyStateMixin` with the interface?
🤔 
**Not a blocker**

Reply:
IMO, no need for `ReadonlyStateMixin` for now. It doesn't have `notify` or any other logic.